exchanges/okgroup/okgroup_types.go

```diff
@@ -319,6 +319,7 @@ type GetSpotOrderResponse struct {
 	Notional       string    `json:"notional"`
 	OrderID        string    `json:"order_id"`
 	Price          float64   `json:"price,string"`
+	PriceAvg       float64   `json:"price_avg,string"`
```

Comment:
Reference: https://www.okex.com/docs/en/#spot-list

price_avg (Average filled price) is a valid field from their spot trading API